test/contracts/channel_htlc.aes

```diff
@@ -96,7 +96,7 @@ contract ChannelHTLC =
 
     let id : hash = generate_id(sender', state.client, amount', hash_lock')
 
-    require(!recieve_exists(id), "Recieve entry already exists")
+    require(!receive_exists(id), "Receive entry already exists")
```

Comment:
You also need to fix the function itself:   `function recieve_exists(id : hash) : bool =` (line 215)